apps/explorer-e2e/src/specs/index.spec.ts

```diff
@@ -3,15 +3,15 @@ import { fillTextInputByName } from '@siafoundation/e2e'
 
 test('homepage', async ({ page }) => {
   await page.goto('/')
-  await expect(page.getByText('siascan')).toBeVisible()
-  await expect(page.getByText('zen')).toBeVisible()
+  await expect(page.getByTestId('explorer-identity-appName')).toBeVisible()
```

Comment:
These are fine but if they start getting very long I think adding multiple short ones can be better as its super clear where the test is targeting and the higher level ones can be reused. eg: `page.getByTestId('navbar').getByTestId('network')`

Reply:
I don't plan on making these any longer. Nor will they be as hierarchal as this may suggest. What I'm going for is truly unique identifiers. Within the app, I can have discipline and make that happen, but something like the `explorer` prefix helps me know that for sure, if I'm disciplined, I will never search a testID and come up with multiple items outside my scope. Please feel free to bring this up again if you feel I'm getting too verbose with them.